### DISCUSSION TITLE
Find a request by route path

### DISCUSSION THREAD

**Discussion by javaherisaber** [Kong/insomnia#4025] [Ideas]
There is a `Filter` option to find requests by name.
It would be more convenient to find the request by route path also.

Example:
Request: `/user/auth/login`
Name: **Login**

In Filter input, we put `/user/auth` to find all auth related requests

**Comment by protono** [comment]
The request filter on the debug screen uses the description of the end points, please make it support the pathname of the endpoints as well.

Example, an endpoint called GET /users with a description of 'retrieve contact names'

Currently entering 'users' in the request filter will not filter the endpoint, but entering 'contact' will
Expected: both 'users' and 'contact' should filter the endpoint
